Remotion/Web/UnitTests/Core/ExecutionEngine/WxeContextTest.cs

```diff
@@ -38,43 +40,42 @@ public class WxeContextTest
   private Type _functionType;
   private string _functionTypeName;
   private string _resource;
+  private NameValueCollection _queryString;
 
   [SetUp]
   public virtual void SetUp ()
   {
-    NameValueCollection queryString = new NameValueCollection();
-    queryString.Add(WxeHandler.Parameters.ReturnUrl, "/Root.wxe");
+    _queryString = new NameValueCollection();
+    _queryString.Add(WxeHandler.Parameters.ReturnUrl, "/Root.wxe");
 
-    _currentHttpContext = WxeContextMock.CreateHttpContext(queryString);
+    _currentHttpContext = WxeContextMock.CreateHttpContext(_queryString);
 
     _functionType = typeof(TestFunction);
     _functionTypeName = TypeUtility.GetPartialAssemblyQualifiedName(_functionType);
     _resource = "~/Test.wxe";
 
     UrlMappingConfiguration.Current.Mappings.Add(new UrlMappingEntry(_functionType, _resource));
 
-    _currentWxeContext = new WxeContextMock(_currentHttpContext, queryString);
+    _currentWxeContext = new WxeContextMock(_currentHttpContext, _queryString, new WxeUrlSettings(), new WxeLifetimeManagementSettings());
     WxeContext.SetCurrent(_currentWxeContext);
 
-    WebConfigurationMock.Current = new Remotion.Web.Configuration.WebConfiguration();
-    WebConfigurationMock.Current.ExecutionEngine.MaximumUrlLength = 100;
-
   }
 
   [TearDown]
   public virtual void TearDown ()
   {
     WebConfigurationMock.Current = null;
-    Remotion.Web.ExecutionEngine.UrlMapping.UrlMappingConfiguration.SetCurrent(null);
+    UrlMappingConfiguration.SetCurrent(null);
   }
 
   [Test]
   public void GetStaticPermanentUrlWithDefaultWxeHandlerWithoutMappingForFunctionType ()
   {
-    WebConfigurationMock.Current = WebConfigurationFactory.GetExecutionEngineWithDefaultWxeHandler();
```

Comment:
Ah, I understand: we had this class in the unittesting helpers assembly. 
#things_I_happily_get_rid_off